app/Controller/CoGroupsController.php

```diff
@@ -681,6 +792,64 @@ public function paginationConditions() {
       $pagcond['conditions']['CoGroup.group_type'] = $searchterm;
     }
 
+    // Exclude admin groups
+    if(!empty($this->params['named']['search.noadmin'])) {
+      $pagcond['conditions']['CoGroup.group_type <>'] = 'A';
+    }
+
+    // Filter by membership and ownership
+    // First get the CoPersonID to work against either the current user, or the user being acted on in select mode
+    $coPersonId = $this->Session->read('Auth.User.co_person_id');
+    if($this->action == 'select') {
+      $coPersonId = filter_var($this->request->params['named']['copersonid'], FILTER_SANITIZE_SPECIAL_CHARS);
+    }
+
+    // If both member and owner are selected, use a single join with "OR"
+    if(!empty($this->params['named']['search.member']) && !empty($this->params['named']['search.owner'])) {
+      $pagcond['joins'][] = array(
```

Comment:
Joins need to be tested against ChangelogBehavior. The easiest way to do this is to make changes in a record of the target table and then make sure you pull the newer values, not the older ones.

Reply:
This appears to be working; changes to member or owner status are immediately reflected in the pagination of the group (and group members) listing. (Tested: changes to member/owner status by an admin. The member immediately saw the change on reload/clicking a tab. Also tested updates to nested groups.)

The "Nested Groups" and "Email Lists" tabs both display the changelog at the bottom of their views, but these only reflect the Group properties changes. I suspect these should be removed so the context is clear. Do we need changelog behavior for these? The "Provisioned Services" has a changelog display too.